Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -76,27 +76,30 @@ def get_current_splunk_time(splunk_service):
 
 def rawToDict(raw):
     result = {}  # type: Dict[str, str]
-    if 'message' in raw:
-        raw = raw.replace('"', '').strip('{').strip('}')
-        key_val_arr = raw.split(",")
-        for key_val in key_val_arr:
-            single_key_val = key_val.split(":")
-            if len(single_key_val) > 1:
-                val = single_key_val[1]
-                key = single_key_val[0].strip()
-
-                if key in result.keys():
-                    result[key] = result[key] + "," + val
-                else:
-                    result[key] = val
+    try:
+        result = json.loads(raw)
+    except ValueError:
+        if 'message' in raw:
+            raw = raw.replace('"', '').strip('{').strip('}')
+            key_val_arr = raw.split(",")
+            for key_val in key_val_arr:
+                single_key_val = key_val.split(":")
```

Comment:
in case the value contains `:` as well
```suggestion
                single_key_val = key_val.split(":", 1)
```